src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -272,6 +272,13 @@ public function beforeSave(Event $event, EntityInterface $entity, ArrayObject $o
         $newOptions = [$this->_translationTable->getAlias() => ['validate' => false]];
         $options['associated'] = $newOptions + $options['associated'];
 
+        // Check early if empty translations are present in the entity.
```

Comment:
May be move this to a protected method where the comments will be the documentation of the method?